<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="customStyleCheck">
     <property name="text">
      <string>Use custom style</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chordPreviewCheck">
     <property name="text">
      <string>Preview with all units</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pauseReseekCheck">
     <property name="text">
      <string>Spacebar stops instead of pauses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="swapScrollOrientationCheck">
     <property name="text">
      <string>Swap vertical / horizontal scrolling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="swapZoomOrientationCheck">
     <property name="text">
      <string>Swap vertical / horizontal zoom</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="velocityDragCheck">
     <property name="text">
      <string>Adjust velocity with vertical drag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoAddUnitCheck">
     <property name="text">
      <string>Auto-add unit with each new voice</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>MIDI input port (work in progress)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="midiInputPortCombo"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
